web/app/components/workflow/nodes/_base/components/variable/output-var-list.tsx

```diff
@@ -64,6 +58,39 @@ const OutputVarList: FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [list, onChange, outputs, outputKeyOrders])
 
+  const handleVarNameFocus = useCallback((index: number) => {
+    return () => {
+      const oldVar = list[index]
+      setEditVar(oldVar)
+    }
+  }, [list])
+
+  const handleVarNameBlur = useCallback((index: number) => {
+    return (e: React.ChangeEvent<HTMLInputElement>) => {
+      const oldKey = list[index].variable
+      const newKey = e.target.value
+      if (list.map(item => item.variable?.trim()).includes(newKey.trim())) {
```

Comment:
When the blur effect is applied, it keeps throwing errors, even when there is only one item.
Perhaps you should filter out its own item.

Reply:
yes, I do it in next commit

fix: filter validate self key